octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -450,27 +453,36 @@ $(function () {
             }
         };
 
-        self._handle_session_expired = function (triggerUrl) {
+        self._handle_session_expired = function (triggerUrl, requestData) {
             if (self.isCurtainOpened > 0) {
                 self.error401Count++;
                 if (!(triggerUrl in self.triggerUrlCount)) {
                     self.triggerUrlCount[triggerUrl] = 0;
                 }
                 self.triggerUrlCount[triggerUrl]++;
+
+                if (!(triggerUrl in self.triggerData)) {
+                    self.triggerData[triggerUrl] = [requestData];
+                }
+                self.triggerData[triggerUrl].push(requestData);
```

Comment:
Change `self.triggerUrlCount` into `self.triggerUrl` without adding `self.triggerData`

```suggestion
                if (!(triggerUrl in self.triggerUrl)) {
                    self.triggerUrl[triggerUrl]['count'] = 0;
                    self.triggerUrl[triggerUrl]['data'] = [requestData];
                }
                self.triggerUrl[triggerUrl]['count']++;
                self.triggerUrl[triggerUrl]['data'].push(requestData);
```